sections/synths/components/SynthPriceCol.tsx

```diff
@@ -32,7 +43,9 @@ const SynthPriceCol: FC<SynthPriceColProps> = ({ currencyKey }) => {
 					currencyKey={currencyKey}
 					price={price}
 					sign={selectedPriceCurrency.sign}
-					change={historicalRates.data?.change}
+					change={historicalRates.data?.[0].rate
+						.div(historicalRates.data?.[historicalRates.data?.length - 1].rate)
+						.toNumber()}
```

Comment:
Can you create a function that takes historicalRates.data and returns change? Then we can add some tests and add some logic to avoid infinity

Reply:
https://github.com/Synthetixio/staking/pull/769/commits/0a9be4a938eb698efd5a4b16244d182fdacbbbb5